README.md

```diff
@@ -84,7 +84,8 @@ The TypeScript typings are included in the package and so all you have to do is:
 
 ```typescript
 import * as Ably from 'ably';
-let options: Ably.Types.ClientOptions = { key: 'foo' };
+
+let options: Ably.Types.ClientOptions = {key: 'foo'};
```

Comment:
You've changed the formatting slightly here
```suggestion
let options: Ably.Types.ClientOptions = { key: 'foo' };
```

Reply:
I really need to get my editor config setup to be inline with the prettier config